methods/array/chunk.js

```diff
@@ -0,0 +1,24 @@
+function chunk(array, size) {
+  if (!(array instanceof Array)) {
```

Comment:
I think check: Array.isArray() ->  is more reliable. You can try it, everywhere basically

Reply:
![image](https://user-images.githubusercontent.com/35407446/218055755-66cf4564-d4f0-4d78-9173-35030972d430.png)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray